### PR TITLE
Add step to clone project

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ RealWorld.io frontend.
 
 You need to have docker, [pipenv](https://pipenv.readthedocs.io/) and Python 3.7 installed on your machine. Docker should be running. Then you can run:
 
+    $ cd ~/projects
```
Comment (suryasr007) [inline readme.md]: This step is user specific

Comment (stevepiercy) [reply readme.md]: Please use [GitHub suggestions (step 6)](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) to suggest an alternative that conveys the intent of "user's projects folder".

Comment (suryasr007) [reply readme.md]: With respect to this/any repository, is it really required to convey "user's projects folder"? 

I would remove the `cd ~/projects` line and proceed with the next two lines.

Comment (stevepiercy) [reply readme.md]: Where would you rather have them put it? Nowhere? Anywhere?

Comment (sayanarijit) [reply readme.md]: Agree with @suryasr007 There's no need to mention `cd ~/projects `specifically. It depends on the user where they want to clone this project. If the user is not familiar with git cloning, then the default path i.e. present working directory would do fine.

Comment (zupo) [reply readme.md]: I agree with Steve on this one. It's better to be more explicit in introductionary docs such as this one.

**Pull request merged**